addons/mail/static/tests/helpers/mock_models.js

```diff
@@ -187,6 +187,12 @@ class MockModels {
                 },
                 records: [],
             },
+            'mail.thread_member': {
```

Comment:
mock server is supposed to simulate the server not the client, so this should be reflecting channel.partner model